### DISCUSSION TITLE
PVR Manager window not opening?

### DISCUSSION THREAD

**Discussion by WilliamOfGaunt** [Ascoware/get-iplayer-automator#478] [General]
Hmmm... Not sure when this started, but I've only just noticed that the PVR Manager window is no longer opening when I click on the PVR button or use the Window menu item.

Oh - now I find that the Show Log and Preferences buttons don't do anything either... What have I done?

Running version 1.26.1 on MacOS Sonoma, 14.4.1, if this helps.

**Comment by Richard546931** [comment]
Before buying MacBook Air or Pro will Automator work on Sonoma OS?

**Comment by kw4c** [comment]
Works for me on an m1 mini running Sonoma.   
  
  
  
  
  
  
  

  
  
>
